app/models/setting/general.rb

```diff
@@ -25,7 +25,8 @@ def self.load_defaults
         self.set('use_gravatar', N_("Foreman will use gravatar to display user icons"), true),
         self.set('db_pending_migration', N_("Should the `foreman-rake db:migrate` be executed on the next run of the installer modules?"), true),
         self.set('db_pending_seed', N_("Should the `foreman-rake db:seed` be executed on the next run of the installer modules?"), true),
-        self.set('proxy_request_timeout', N_("Max timeout for REST client requests to smart-proxy"), 60)
+        self.set('proxy_request_timeout', N_("Max timeout for REST client requests to smart-proxy"), 60),
+        self.set('colorize_logging', N_("Should ANSI color codes be used when logging information in production"), false)
```

Comment:
not really, we'd need more systematic solution anyway so ignore it here... in future I could imagine icon indicating that restart is needed after change but that's out of scope of this PR